core/scripts/vrfv2/testnet/v2scripts/super_scripts.go

```diff
@@ -273,6 +285,25 @@ func VRFV2DeployUniverse(
 	helpers.PanicErr(err)
 	fmt.Printf("Subscription %+v\n", s)
 
+	if deployVRFOwner {
+		// VRF Owner
+		vrfOwner, err := vrf_owner.NewVRFOwner(vrfOwnerAddress, e.Ec)
+		helpers.PanicErr(err)
+		authorizedSendersSlice := helpers.ParseAddressSlice(vrfKeyRegistrationConfig.RegisterAgainstAddress)
```

Comment:
here we should use `vrfPrimaryNodeSendingKeys` which is set from `vrf-primary-node-sending-keys`.

`RegisterAgainstAddress` is just one address which is used to register proving key against. from this address you can perform `coordinator.oracleWithdraw` to withdraw earned funds from rand request fulfilments. It can be the sending keys of the nodes, but its much easier to specify an address to which you have access and perform `oracleWithdraw`

Reply:
Fixed this to use `nodesMap[model.VRFPrimaryNodeName].SendingKeys[0].Address`

Reply:
but why just one key, we need to perform  `vrfOwner.SetAuthorizedSenders` for all sending keys